### PR TITLE
vert 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by apeschel**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by apeschel** [comment]
If you're wondering why I am adding this formula, it's because this tool is being used by this repository for testing: https://github.com/kubernetes/charts

File: Formula/vert.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    cd "src/github.com/Masterminds/vert" do
+      system "dep", "ensure"
+      system "make", "build"
+      bin.install "vert"
+    end
+  end
+
+  test do
+    output = shell_output("#{bin}/vert --version")
+    assert_match "vert version 0.1.0", output
```
Comment (dunn) [inline Formula/vert.rb]: If you've got a functional test, this version check shouldn't be necessary.

File: Formula/vert.rb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  depends_on "dep" => :build
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/Masterminds/vert").install buildpath.children
+    cd "src/github.com/Masterminds/vert" do
+      system "dep", "ensure"
+      system "make", "build"
+      bin.install "vert"
```
Comment (DomT4) [inline Formula/vert.rb]: Would be surprised if a `prefix.install_metafiles` line was not required inside this block.

Comment (apeschel) [reply Formula/vert.rb]: @DomT4 What does `prefix.install_metafiles` do? I'm not finding it in the docs.

Comment (apeschel) [reply Formula/vert.rb]: It looks like it copies the README and LICENSE files?

Comment (dunn) [reply Formula/vert.rb]: Yep!  It's run automatically in the root directory, so we only need to call it manually when everything is in a subdirectory.

**Comment by DomT4** [comment]
Apart from the notability issue, I'm not sure there's much else for OP to do here to be fair.

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

**Comment by ilovezfs** [comment]
Ignoring
```
vert:
  * GitHub repository not notable enough (<20 forks, <20 watchers and <50 stars)
```

**Review by ilovezfs — APPROVED** [review]

**Comment by ilovezfs** [comment]
Thanks for the :new: formula and your first contribution to Homebrew @apeschel!

**Pull request closed without merging**